packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -4841,5 +4856,284 @@ describe("database test", async () => {
         ).rejects.toThrowError(eServiceNotFound(eservice.id));
       });
     });
+    describe("create risk analysis", () => {
```

Comment:
Could you move this test inside `describe("Catalog service", () => {...})`? (instead of  `describe("ReadModel Service", () => {...})`)